bindings/pydeck/pydeck/bindings/types.py

```diff
@@ -0,0 +1,101 @@
+import abc
+import json
+from collections.abc import Iterable
+
+TYPE_IDENTIFIER = "@@type"
+FUNCTION_IDENTIFIER = "@@="
+ENUM_IDENTIFIER = "@@#"
+
+
+class Type(metaclass=abc.ABCMeta):
+    """Base class for Pydeck defined type"""
```

Comment:
Nit: pydeck is lower-cased, like [pandas](https://pandas.pydata.org/docs/) or [folium](https://python-visualization.github.io/folium/).